src/controllers/category.js

```
@@ -0,0 +1,17 @@
+const categoryService = require('~/services/category')
+
+const getCategories = async (req, res) => {
+  const { search = '', page = 1, limit = 20 } = req.query;
+
+  const categories = await categoryService.getCategories({
+    search,
+    page: parseInt(page),
+    limit: parseInt(limit),
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)